src/ContextVeil.php

```diff
@@ -57,9 +45,7 @@ public function __construct(
      */
     public function __call(string $name, array $arguments): mixed
     {
-        if (isset($this->methods[$name])) {
-            $this->origin = ($this->context)($this->origin);
-        }
+        $this->origin = $this->reaction->edited($this->origin, $name);
         return $this->origin->$name(...$arguments);
     }
```

Comment:
Now always passes a stored instance to `Reaction` and then returns method result.